pkg/arvo/lib/pull-hook.hoon

```diff
@@ -349,13 +350,16 @@
       :-  -:!>(*(unit path)) 
       ?:(?=(%0 -.res) p.res ~)
     =?  failed-kicks  !?=(%0 -.res)
-      =/  tang
+      =/  =tang
         :+  leaf+"failed kick handler, please report" 
           leaf+"{<rid>} in {(trip dap.bowl)}"
         ?:  ?=(%2 -.res)
           p.res
         ?>  ?=(%1 -.res)
-        (turn `(list *)`p.res (cork path smyt))
+        =/  paths=(unit (list path))  ((soft (list path)) p.res)
+        ?~  paths  ~
+        %+  turn  u.paths
+        (cork path smyt)
```

Comment:
This is no longer an untyped list of paths, it's a single untyped path (see `$toon`).

Reply:
Latest commit addresses that. LMK what you think.